MapView/Map/RMConfiguration.h

```diff
@@ -47,6 +47,9 @@
 /** A Mapbox API access token. Obtain an access token on your [Mapbox account page](https://www.mapbox.com/account/apps/). */
 @property (nonatomic, retain) NSString *accessToken;
 
+/** A default URL session for the framework */
+@property (nonatomic, readonly) NSURLSession *mapBoxUrlSession;
```

Comment:
I think I'd just call this `urlSession` here. It's an internal class, so it doesn't really need to be namespaced. You can always add an identifier to the session configuration if you need it for debugging. https://developer.apple.com/reference/foundation/nsurlsessionconfiguration